.clang-tidy

```diff
@@ -50,7 +49,6 @@ misc-header-include-cycle,
 misc-misleading-identifier,
 misc-redundant-expression,
 misc-unused-parameters,
-performance-no-int-to-ptr,
```

Comment:
This is often needed in an OS, so I'll just remove it entirely. Maybe when Qodana for C/C++ is more stable and has its own config, I can find ways to exclude kernel from this rule.